tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-continue-dialog.html

```diff
@@ -0,0 +1,193 @@
+          class="go-button"
+          icon="arrow-forward"
+          title="Tensor Condition Go"
+          on-tap="_tensorContinueGoButtonCallback">
+        </paper-icon-button>
+        <paper-input
+          id="ref-value"
+          class="input-box"
+          label="Reference value to compare to"
+          type="number"
+          value="{{_tensorConditionRefValue}}"
+          hidden="[[_isRefValueInputHidden]]">
+        </paper-input>
+      </div>
+    </paper-dialog>
+    <style include="dashboard-style"></style>
+    <style>
+      :host .continue-to-type-name {
+        font-weight: bold;
+      }
+      :host paper-dialog {
+        width: 36vw;
+      }
+      :host .input-box {
+        display: inline-block;
+        position: relative;
+        width: 80%;
+        font-size: 110%;
+      }
+      :host .go-button {
+        position: relative;
+        width: 15%;
+        display: inline-block;
+      }
+    </style>
+  </template>
+  <script src="health-pills.js"></script>
+  <script>
+    Polymer({
+      is: 'tf-debugger-continue-dialog',
+      properties: {
+        continueNum: {  // How many times to continue over, e.g., Session.Runs.
+          type: Number,
+          value: 5,
+        },
+
+        // Callback for the session-runs go button.
+        sessionRunGo: Function,
+
+        // Handler for tensor-condition go, invoked by the callback of the
+        // tensor-condition go button.
+        // Signature of this function:
+        // Args:
+        //   conditionKey: Key of the tensor-value condition (e.g., NAN_OR_INF),
+        //     see health-pills.ts for more details.
+        //   refValue: Optional reference value. It is required by some of the
+        //     tensor-value conditions (e.g., STDDEV_GT).
+        tensorConditionGo: Function,
+
+        _selectedTensorCondition: String,
+
+        _tensorConditionRefValue: {
+          type: Number,
+          value: 0,
+          notify: true,
+        },
+
+        _isRefValueInputHidden: {
+          type: Boolean,
+          value: true,
+          notify: true,
+        }
+      },
+
+      observers: [
+        "_onSelectedTensorConditionChanged(_selectedTensorCondition)",
+      ],
+
+
+      open() {
+        this.$.continueDialog.open();
+      },
+
+      close() {
+        this.$.continueDialog.close();
+      },
+
+      _sessionRunGoButtonCallback() {
+        this.sessionRunGo();
+      },
+
+      _tensorContinueGoButtonCallback() {
+        if (this._selectedTensorCondition == null) {
+          return;
+        }
+        const tensorConditionKey =
+            tf_debugger_dashboard.tensorConditionDescription2Key(this._selectedTensorCondition);
+        if (tensorConditionKey == null) {
+          console.error(
+              'Invalid Tensor Condition name:' + this._selectedTensorCondition);
+        }
+        // Validate that this._tensorConditionRefValue is a number.
+        if (isNaN(this._tensorConditionRefValue)) {
```

Comment:
You can use `if !(_.isFinite(this._tensorConditionRefValue))`. That also checks for no Infinity.